.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5.1.0
```

Comment:
can we update to v5.3.0 while we're here?

Reply:
Thanks @jackfrancis for taking a look, I will update this PR.

Reply:
@jackfrancis, I have addressed your comment.